internal/embedding/data_test.go

```diff
@@ -6,27 +6,28 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/assert"
```

Comment:
curious why change from require to assert? they are basically the same anyways

Reply:
Require stops the execution of the test immediately so IMO it should be used for things in the test set up.

Assert doesn't stop the execution of the test so IMO it should be used for the assertions at the end of the test. This way a developer can see multiple test assertion failures (i.e. from different test cases)